web/tests/test_views.py

```
@@ -532,6 +532,22 @@ def test_calendar_display(self):
         session_date = self.future_session.start_time.date()
 
+        # Debug information
+        print(f"Future session date: {session_date}")
+        print(f"Calendar context: {calendar_weeks}")
+
+        # Make sure the current month in the calendar matches the session month
+        current_month = response.context["current_month"]
+        if current_month.month != session_date.month or current_month.year != session_date.year:
+            print(
+                f"Calendar month ({current_month.month}/{current_month.year}) "
+                f"doesn't match session date month ({session_date.month}/{session_date.year})"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)